src/libs/convertToLTRForComposer/index.android.ts

```diff
@@ -1,8 +1,62 @@
+import CONST from '@src/CONST';
 import ConvertToLTRForComposer from './types';
 
+/**
+ * Android only - The composer can be converted to LTR if its content is the LTR character followed by an @ or space
+ */
+function canComposerBeConvertedToLTR(text: string): boolean {
+    // this handle cases when user type only spaces
+    const containOnlySpaces = /^\s*$/;
+    // this handle the case where someone has RTL enabled and they began typing an @mention for someone.
+    const startsWithLTRAndAt = new RegExp(`^${CONST.UNICODE.LTR}@$`);
+    // this handle cases could send empty messages when composer is multiline
```

Comment:
I'm not quite sure what this means and the grammar is a bit confusing. Are you trying to say this?

> This regex handles the case where the composer can contain multiple lines of whitespace.

Reply:
That's right, sorry 😣!